### PR TITLE
Have commit validation accept different cases for keywords

### PR DESCRIPTION
**Pull request by daviddavis**
fixes #7404

### CODE REVIEW

File: templates/travis/.travis/validate_commit_message.py.j2 [hunk 1]
```diff
@@ -51,7 +51,7 @@ print("Checking commit message for {sha}.".format(sha=sha[0:7]))
 regex = r"(?:{keywords})[\s:]+#(\d+)".format(keywords=("|").join(KEYWORDS))
 pattern = re.compile(regex)
 
-issues = pattern.findall(message)
+issues = pattern.findall(message, flags=re.IGNORECASE)
```
Comment (fao89) [inline templates/travis/.travis/validate_commit_message.py.j2:54]: GHA did not like it:
```
TypeError: 'flags' is an invalid keyword argument for findall()
```
https://github.com/pulp/plugin_template/pull/256/checks?check_run_id=1038033183#step:7:23

**Comment by dralley** [comment]
@daviddavis Any blocker to merge?

**Comment by mdellweg** [comment]
I think, the only question was, whether the communication with pulp.plan.io was also working with capitalized keywords.
If that is still true, we could add a `tolower` there.

**Comment by daviddavis** [comment]
Looks like based on [this test](https://www.redmine.org/projects/redmine/repository/revisions/2637/entry/trunk/test/unit/changeset_test.rb#L26) that keyword checking is case insensitive for redmine. Merging.

**Pull request merged**